vars/mtaBuild.groovy

```diff
@@ -1,6 +1,9 @@
 import com.sap.piper.ConfigurationLoader
 import com.sap.piper.ConfigurationMerger
 
+import groovy.transform.Field
+
+@Field DEFAULT_MTA_JAR_NAME = 'mta.jar'
```

Comment:
`def`, no `def`? In neo it is different.

Reply:
done, changed to ```def```